src/ray/raylet/scheduling/local_task_manager_interface.h

```diff
@@ -71,5 +71,66 @@ class ILocalTaskManager {
   virtual size_t GetNumWaitingTaskSpilled() const = 0;
   virtual size_t GetNumUnschedulableTaskSpilled() const = 0;
 };
+
+/// A noop local task manager, which is used by gcs node only.
```

Comment:
```suggestion
/// A noop local task manager. It is a no-op class. We need this because there's no "LocalManager" when the `ClusterTaskManager` is used within GCS. In the long term, we should make `ClusterTaskManager` not aware of `LocalTaskManager`
```

Reply:
Sure.